### PR TITLE
Update Linux AppData description

### LINKED ISSUE #2717: Linux Feature description
### Are you using the latest Dosbox-Staging Version?

- [X] I have checked releases and am using the latest release.

### Different version than latest?

devel

### What Operating System are you using?

Linux x86_64

### Whats your question and how can we help?

The `contrib/linux/dosbox-staging.metainfo.xml` file should be updated before the next release to reflect the current state of the project. I thought to simply add voodoo and reelmagic, but there are probably various other parts that should be updated, such as the video outputs.

So I thought lets ask the question, what should be added/removed.

For those that don't know, this file is used by the graphical software installer used on most modern Linux distributions, and also used by [flatpak](https://flathub.org/apps/io.github.dosbox-staging)

The specification for the file format can be found here: https://www.freedesktop.org/software/appstream/docs/

### Code of Conduct & Contributing Guidelines

- [X] Yes, I agree.

### PR DESCRIPTION
**Pull request by rderooy**
Simplify description. Remove feature list as discussed, to prevent having to update the description for each release.

Fixes #2717

### CODE REVIEW

File: contrib/linux/dosbox-staging.metainfo.xml [hunk 1]
```diff
@@ -5,31 +5,14 @@
...
-          scaling, sharp-bilinear scaling, OpenGL shaders, and more</li>
-    </ul>
-    <p>
-      DOSBox Staging is highly configurable and sufficiently-optimized to run
-      any DOS game on a modern computer.
+      DOSBox Staging is a modern continuation of DOSBox with advanced features and current development practices. It emulates an Intel x86 PC running the MS-DOS operating system with the primary aim of running games, but most application programs will work as well.
+
+      It is meant as a (mostly) drop-in replacement for older DOSBox versions. Your existing configurations will continue to work, but you will have access to many advanced features.
+
+      For the full list of supported features, please visit the DOSBox Staging website.
```
Comment (johnnovak) [inline contrib/linux/dosbox-staging.metainfo.xml:15]: Don't we need a link to the website here?

**Comment by rderooy** [comment]
Your not supposed to put links in the description. If you look for instance at the flatpak page, you will see there are buttons for homepage and bugtracker, which are defined in the XML.

Here are the supported URL's:
https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#spec-component-filespec

The definition allows for the following URL types:
- homepage
- bugtracker
- faq
- help
- donation
- translate
- contact
- vcs-browser
- contribute.

But not every tenderer may necessarily show all URLs.

File: contrib/linux/dosbox-staging.metainfo.xml (hunk 1 above)
Comment (kcgen) [reply contrib/linux/dosbox-staging.metainfo.xml:15]: Thanks for the explanation (below: https://github.com/dosbox-staging/dosbox-staging/pull/2732#issuecomment-1666494042), @rderooy :+1:

**Review by kcgen — APPROVED** [review]

**Comment by johnnovak** [comment]
> Your not supposed to put links in the description. If you look for instance at the flatpak page, you will see there are buttons for homepage and bugtracker, which are defined in the XML.

Makes sense, and that's what I suspected @rderooy. I guess the URL must be defined in another file then because I can't find it in this one.

**Comment by rderooy** [comment]
> > Your not supposed to put links in the description. If you look for instance at the flatpak page, you will see there are buttons for homepage and bugtracker, which are defined in the XML.
> 
> Makes sense, and that's what I suspected @rderooy. I guess the URL must be defined in another file then because I can't find it in this one.

I'm not following. Towards the end of the file there are two URLs.

```
  <url type="homepage">https://dosbox-staging.github.io/</url>
  <url type="bugtracker">https://github.com/dosbox-staging/dosbox-staging/issues</url>
```

**Comment by johnnovak** [comment]
> > Makes sense, and that's what I suspected @rderooy. I guess the URL must be defined in another file then because I can't find it in this one.
> 
> I'm not following. Towards the end of the file there are two URLs.
> 
> ```
>   <url type="homepage">https://dosbox-staging.github.io/</url>
>   <url type="bugtracker">https://github.com/dosbox-staging/dosbox-staging/issues</url>
> ```

Oh right, I did not press the little arrow to see the rest of the file 😄 Cheers!

**Pull request merged**